algorithms/maths/modular_Exponential.py

```diff
@@ -0,0 +1,15 @@
+def modular_Exponential(base, power, mod):
```

Comment:
please use `snake_case`. change the function name to -> `modular_exponentiation()` and reflect them in the READMEs and tests.